app/assets/stylesheets/nucore.scss

```diff
@@ -139,6 +139,10 @@ li.nav-spacer {
   margin-bottom: 0;
 }
 
+.alert {
+  color: #24506B; // minimum lightness for WCAG AAA compliance (accessibility).
```

Comment:
I think we may want a slightly different color between `alert-info` and `alert-danger`. We might even want to think of only using the alerts for flashes. I think we use it a couple other ways including the welcome text on the homepage.